api/DTOs/RegionDTO.cs

```diff
@@ -1,12 +1,11 @@
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
-
 namespace api.DTOs
 {
     public class RegioDTO
```

Comment:
Region...

Reply:
in line 8 the name is RegionDTO instead of RegioDTO

Reply:
Resolve